spec/system/denounce_spec.rb

```diff
@@ -0,0 +1,109 @@
+require 'rails_helper'
+
+RSpec.describe 'Denounce', type: :system do
+  before do
+    driven_by(:selenium)
+  end
+
+  let :user do
+    FactoryBot.create(:user)
+  end
+  describe 'accessing my denounces' do
+    context 'when user is not signed in' do
+      it 'redirects to sign in' do
+        visit('/')
+        click_on('My denounces')
+        expect(page).to have_content(/You need to sign in/)
+      end
+    end
+    context 'when user is signed in' do
+      it 'list my denounces' do
+        FactoryBot.create_list(:denounce, 6, user: user)
+        login(as: user)
+        click_on('My denounces')
+        expect(page).to have_selector('#accordionDenounces .accordion-item', count: 6)
+      end
+    end
+  end
+
+  describe 'Show denounce' do
+    it 'displays denounce details' do
+      FactoryBot.create(:denounce, title: 'Laudantium dicta illo', user: user)
```

Comment:
move this into a memoized helper method, so you can reuse in all your specs instead of repeating in each spec https://relishapp.com/rspec/rspec-core/v/3-4/docs/helper-methods/let-and-let

Reply:
See: https://github.com/BC-MAR-21-ROR/proyecto-denuncia-ciudadana-proyecto-team02/pull/16/commits/8d197f0c3db5de72e22e1517db6d96ced2ba97d6